Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
 import './Card.css';
+import CardImg from './CardBanner';
+import CardContent from './CardContent';
+
+const CardContainer = () => {
+    return (
+        <div onClick={myFunction} className="card-wrapper">
```

Comment:
Good job on making your entire card like a link and implementing your own function on line 15.